examples/training/classification/densenet.py

```diff
@@ -36,10 +37,24 @@
 )
 
 NUM_CLASSES = 10
-EPOCHS = 10
+EPOCHS = 250
 BATCH_SIZE = 32
 WEIGHTS_PATH = "weights.hdf5"
 
+AUGMENT_LAYERS = [
+    keras_cv.layers.RandAugment(value_range=(0, 255), magnitude=0.7),
+    keras_cv.layers.CutMix(),
+    keras_cv.layers.MixUp(),
+    keras_cv.layers.RandomFlip(),
```

Comment:
believe it or not, order matters.

RandomFlip() => first
RandAugment() => second
RandomCutout() => third
CutMix() => fourth
MixUp => fifth

Reply:
Good to know -- is there a good reference I can look at about why order is important here?

Reply:
(Such a reference would probably be good to point to in the training script)

Reply:
not that I know of actually... 🤔